ts/packages/cli/src/base.ts

```diff
@@ -198,14 +200,15 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
     this.config.UNICODE_NULL = '\u0000';
 
     await super.init();
-    const { args, flags } = await this.parse({
+    const { args, flags, raw } = await this.parse({
       flags: this.ctor.flags,
       baseFlags: (super.ctor as typeof BaseCommand).baseFlags,
       args: this.ctor.args,
       strict: this.ctor.strict,
     });
     this.flags = flags as Flags<T>;
     this.args = args as Args<T>;
+    this.rawFlags = raw as FlagToken[];
```

Comment:
I need some clarification regarding the decision to use the `FlagToken` type. The type of `raw` is `ParsingToken[]` and casting it to `FlagToken[]` seems dangerous. Is there something that guarantees that the tokens aren't of type `ArgToken`?

Reply:
There are no ArgToken for export command.
This is what will happen if you try to use one
![image](https://github.com/configu/configu/assets/129258356/7e2511a0-39c3-418d-8dd6-6bdf3e5186b2)